WordPress/build.gradle

```diff
@@ -80,7 +80,6 @@ android {
         buildConfigField "boolean", "LIKES_ENHANCEMENTS", "false"
         buildConfigField "boolean", "IS_JETPACK_APP", "false"
         buildConfigField "String", "TRACKS_EVENT_PREFIX", '"wpandroid_"'
-        buildConfigField "String", "PUSH_NOTIFICATIONS_APP_KEY", '"org.wordpress.android"'
```

Comment:
According to CI this is still used on `/home/circleci/project/WordPress/src/main/java/org/wordpress/android/ui/notifications/utils/NotificationsUtils.java:105: error: cannot find symbol`